### DISCUSSION TITLE
Flutter Quickstart: Profile Photo Problem

### DISCUSSION THREAD

**Discussion by samuele-besoli-mndl** [supabase/supabase#4921] [Questions]
Hello, I settled up a flutter app following the quick start but there is a problem with profile photo.
The problem is with the image in avatar because url doesn't contain the token needed to get the image.

Am I missing something?

**Comment by dshukertjr** [comment]
Hi @Sam-MNDL !

Tokens are not needed for the image to be loaded actually. 

In [Set up the database schema](https://supabase.com/docs/guides/with-flutter#set-up-the-database-schema) section of the quick starter guide, you are running the following SQL:

```sql
insert into storage.buckets (id, name, public)
values ('avatars', 'avatars', true);
```
which sets the `public` field of your storage bucket to `true`. This enables Supabase storage to create a publicly accessible url for objects within a storage bucket. 

Could you double check if the `public` field is actually set to `true` within your storage bucket? If you have the orange `Public` label on your storage bucket like the following image, you should be good. 

<img width="726" alt="Screen Shot 2022-01-14 at 12 37 50" src="https://user-images.githubusercontent.com/18113850/149447449-441036f9-93a7-45f3-8e00-c6f994e51bed.png">

**Reply by samuele-besoli-mndl** [reply]
Hello thank you so much for response.

I followed quick start and ran the sql but i dont know why the ​bucket wasn't public.
Now I manually turned public and it works.

For testing out I will retry that sql in a new project.

Thanks you
